ts/Core/Axis/Axis.ts

```diff
@@ -4824,6 +4824,11 @@ class Axis {
             // Record the last value
             lastPos = pos;
         }
+
+        if (pos > roundedMax && tickPositions[tickPositions.length - 1] < roundedMax) {
+            tickPositions.push(roundedMax);
```

Comment:
Is this block necessary? What is the fail case?

Reply:
There would be no 0 tick in the big negative case so the top gets cut off, since the last `pos + tickInterval` is greater than `roundedMax` which is 0.